oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -174,6 +174,20 @@ considers the timing of the next `Open edX named release`_.
 * **Accepted** on Day 14 *(depending on influx of feedback)*
 * **Deprecated/Removing/Removed** - from Day 15 onwards *(depending on resources and technology being removed)*
 
+Consider choosing deprecation and removal dates that allow for a full
+release cycle for transition planning. For example, deprecation might
+occur before Maple named release is finalized, and removal might occur
+afterwards. In this example, a deployment using named releases would
+have a number of months to adjust their Maple deployment to stop
+depending on the deprecated feature before Nutmeg fully removes
+it. This approach would be most appropriate for features that can be
+left in place for an extended period before removal and where a
+transition to an alternative would require a moderate to large amount
+of effort. For more trivial deprecations, it may be appropriate to
+simply deprecate and remove within the same release cycle.
+
+Either way, deprecations and removals should be included in the named release's release notes.
```

Comment:
Note to self, feedback from Diana: "if we want to require adding deprecations into the release notes, we should put that line in the Removing/Removed section" -- plan on making this change.

Reply:
As the release notes writer for the past two named releases, I definitely tried to include deprecation information. For the most part, I picked it up from reading the named release page in the Confluence wiki, and by skimming through DEPR tickets. I'd like to think some details could be extracted from commit messages as well. 

What do you all have in mind?

Reply:
I did a bit of a rewrite and included a note in the Accepted section. Does this seem about right?